aslam_cv/aslam_cv_backend/include/aslam/backend/CameraDesignVariable.hpp

```diff
@@ -42,6 +47,14 @@ namespace aslam {
 
 		boost::shared_ptr<camera_t> camera() { return _camera; }
 
+		/// \brief Get the keypoint time as an expression. If the shutter
+		///        parameters are being estimated, this will be hooked up
+		///        to the camera design variable
+		backend::ScalarExpression keypointTime(const aslam::Time & imageStamp, const Eigen::VectorXd & y);
```

Comment:
Not sure if it makes sense, as it mimics the interfaces of the cameras themselves. As any camera model in the aslam_cv package has the potential to have dynamic values for the keypoint time, I would leave it in the generic CameraDesignVariable.